manager/orchestrator/global.go

```diff
@@ -192,13 +192,20 @@ func (g *GlobalOrchestrator) reconcileOneService(ctx context.Context, service *a
 		log.G(ctx).WithError(err).Errorf("global orchestrator: reconcileOneService failed finding tasks")
 		return
 	}
+	// indicator if service mode was changed to global
+	modeChanged := false
 	// a node may have completed this service
 	nodeCompleted := make(map[string]struct{})
 	// nodeID -> task list
 	nodeTasks := make(map[string][]*api.Task)
 
 	for _, t := range tasks {
 		if isTaskRunning(t) {
+			// tasks from global service has Slot number 0
```

Comment:
I'm going to point out the issue of not using slots for global again :P

The day we add cron with a single slot (0), this is going to fail.